src/applications/vaos/covid-19-vaccine/redux/actions.js

```diff
@@ -260,7 +266,18 @@ export function updateFacilitySortMethod(sortMethod, uiSchema) {
   };
 }
 
-export function getAppointmentSlots(startDate, endDate, initialFetch = false) {
+export function getAppointmentSlots(start, end, initialFetch = false) {
+  let startDate = start;
+  let endDate = end;
+
+  if (!isDate(start)) {
+    startDate = parseISO(start);
+  }
+
+  if (!isDate(end)) {
+    endDate = parseISO(end);
+  }
```

Comment:
This addresses the use case where the user clicks the next button on the calendar view. The date will be in this format,  'yyyy-MM-dd'.